src/modules/ModuleGovernance/query.proposal.ts

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from '@vue/apollo-composable'
+import gql from 'graphql-tag'
+import { Ref } from 'vue'
+import { Address } from '@/core/kaikas'
+import { ProposalState } from './types'
+
+export interface ProposalQueryResult {
+  proposal: {
+    id: Address
+    title: string
+    start: number
+    end: number
+    state: ProposalState
+    choices: string[]
```

Comment:
Couldn't `choices` by typed stricter? Does it have a limited set of values?

Reply:
No, it doesn't. In Snapshot, you are able to add any amount of choices with any names.

Reply:
I see